src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -324,12 +333,20 @@ function MoneyRequestView({
         </OfflineWithFeedback>
     );
 
+    const shouldShowReceiptEmptyState = !hasReceipt && (canEditReceipt || isAdmin || isApprover);
+    /* eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing */
+    const shouldShowMapOrReceipt = showMapAsImage || hasReceipt;
```

Comment:
- Let's move `const showMapAsImage = isDistanceRequest && hasPendingWaypoints;` above this line
- Let's move `const hasPendingWaypoints = transaction?.pendingFields?.waypoints;` above that ^ line and make it `const hasPendingWaypoints = Boolean(transaction?.pendingFields?.waypoints;)`
	- That way `hasPendingWaypoints` is a bool
	- And we can remove the eslint rule suppression